libs/test-setup/src/connectors.rs

```diff
@@ -18,7 +18,7 @@ fn connector_names() -> Vec<(&'static str, BitFlags<Tags>)> {
         ("postgres9", Tags::Postgres.into()),
         ("postgres", Tags::Postgres.into()),
         ("postgres11", Tags::Postgres.into()),
-        ("postgres12", Tags::Postgres.into()),
+        ("postgres12", Tags::Postgres | Tags::Postgres12),
```

Comment:
?

Reply:
from slack:

it's because i needed to do some url manipulation inside the test, to use the user I just created





5:45
so I had to use a concrete url builder function
5:45
didn't feel like making everything generic